assets/queries/cloudFormation/sbd_domain_declared_as_a_resource/test/positive.yaml

```diff
@@ -0,0 +1,11 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "SDB Domain declared"
+Resources:
+  HostedZone:
+    Type: AWS::Route53::HostedZone
+    Properties:
+      Name: "HostedZone"
+  SBDDomain:
+    Type: AWS::SDB::Domain
+    Properties: 
+      Description: "Some information"
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline